### ISSUE TITLE
duel screen on some tablets, adds offset to brush position

### ISSUE THREAD

**Issue by Jacob-Mowat** [mypaint/mypaint#243]
When I use my graphics tablet in duel screen mode, the brush is half of my screen away from where I draw on my graphics tablet( the brush has a weird offset on duel monitors).

**Comment by odysseywestra** [comment]
Could you give us your system information and resolution of your monitors?

**Comment by achadwick** [comment]
@Jacob-Mowat it'd be helpful to know whether GIMP is affected too.

Please let us know your operating system, the version of MyPaint that you're using (check the about box, or paste in the git revision), how your monitors are arranged and their resolutions, and what desktop environment you are using. Without this information we can't do much about this issue.

(But extrapolating wildly, this *may* be a configuration issue. For some drivers and some systems, you have to assign the tablet to one or other monitor.)

**Comment by Jacob-Mowat** [comment]
Os: Windows 8.1 pro
My Paint version: 1.0.0
Graphics Tablet: UGEE m708
monitors:
        - Monitor 1
          Resolution: 1440x900px ( set as main monitor )
        - Monitor 2
          Resolution: 1366x768px ( this is my laptop display )

Driver:
My driver for my tablet has the option of using each monitor ( monitor 1, monitor 2 or both ). NOTE; this problem is only happening with MyPaint, software such as Paint.NET and paint, and others work fine.

**Comment by iirelu** [comment]
@Jacob-Mowat Did you test GIMP, as suggested?

**Comment by Jacob-Mowat** [comment]
@iirelu Yes it works with GIMP, however the pressure isn't working, but I'm sure that's just something that I haven't configured in GIMP.

**Comment by suarezsj** [comment]
Hi!
I have the same issue, but I am working with Autodesk SketchBook Pro 2015 7.0.2
I have a tablt Ugee M708, working on double screen mode:
Os: Windows 8.1 pro
My Paint version: 1.0.0
Graphics Tablet: UGEE m708
monitors:
- Monitor 1 Compaq W220q
Resolution: 1680x1050px (set as main monitor)
- Monitor 2
Resolution: 1366x768px (laptop display)

The only way to solve it is to disable the second screen; then the offset is automatically adjusted to real position of the cursor. I tried several different configurations, but problem persists. 
Any idea about how to fix it? 
Thanks!!

**Comment by achadwick** [comment]
@suarezsj We do not support Sketchbook Pro here, sorry. I didn't see that detail at first, hence the label confusion above. This is the bug/issue tracker for MyPaint, a different program.

@Jacob-Mowat 1.0 is too old for us to support here, sorry. Please update to 1.2.0-alpha and then let us know if the (hopefully improved!) libgdk it uses does not fix the issue for you. Please raise a separate issue if you do.

**Comment by Jacob-Mowat** [comment]
You say 1.2.0-alpha is out ? is there a pre-compiled version of 1.2.0-alpha ?

**Comment by achadwick** [comment]
This may be recurring in MyPaint 1.2.0-beta.1: #401. I have a strong suspicion that it's an upstream issue though.

@Jacob-Mowat sorry, didn't see your response there. There sure is! MyPaint betas can be downloaded form the releases page here on Github. There are more details over at <http://mypaint.intilinux.com/?page_id=6>. Hope that helps!

**Comment by AtsusaKaneytza** [comment]
I'm having the same issue on Windows 8.1.
MyPaint 1.2.0-beta.0+gitexport.adc9031
I'm using a Yiynova MVP10U screen tablet. The pressure sensitivity works, but its also offset when in dual screen.
In GIMP 2.8 32-bit, the pen pressure works fine with my tablet, and doesn't offset when using 2 screens (which is really cool info, thanks btw)

**Comment by AtsusaKaneytza** [comment]
UPDATE:
Apparently, when I set my tablet monitor to "Extend" MyPaint will paint with no offset! That's really weird o_o

**Comment by sebastien-bratieres** [comment]
In my case, when on "Extend these displays", I get the offset, whichever monitor is set as the main display. When on "Duplicate", the offset disappears.

**Comment by AtsusaKaneytza** [comment]
[MyPaint 1.2.0-beta.0+gitexport.adc9031; Windows 8.1, 64-bit; Yiynova MVP10U]
*UPDATE no. 2*
I figured out to draw in dual screen mode with my tablet screen as the main screen (meaning drawing on the tablet will draw in the exact same position on screen.)
*HOWEVER* I only know this works because I've used it in 2 other programs that I own: FireAlpaca/Medibang Paint Pro (they are sister softwares, so there's not much different between them.)
@achadwick I know you asked in #410  - GIMP 2.8 (32-bit) and MyPaint have identical problems with this offset cursor in dual screen, when the tablet monitor is set as the main monitor.
In GIMP 2.8, the program doesn't respond properly to tablet clicks in Screen mode, drawing no pen strokes.
Switching the Pointer mode to "Window" still has the offset, but at least has correct pen pressure.
Drawing with the pointer set to "disabled" will respond with the correct cursor position, but with no pen pressure (as one would expect.)
